### PR TITLE
#591 Internship Page Feedback

### LINKED ISSUE #591: 💄 Internship Page Feedback
Feedback from @adamcogan and @pierssinclairssw for the internship page

https://www.ssw.com.au/training/fullstack

```[tasklist]
### Tasks
- [x] Add breadcrumbs to Training page
- [x] Add training page link to megamenu  ![MicrosoftTeams-image](https://user-images.githubusercontent.com/25432120/236741675-b840b184-66e3-45db-8a44-498e42f15cc5.png)
- [x] Increase font size for the "Who, What, When" section  ![image](https://user-images.githubusercontent.com/25432120/236741951-eb618d0e-97ca-4cff-be91-91f466c7349d.png)
- [x] Fix header text capitalisation (e.g. from "Learn How To Be An" to "Learn how to be an")
- [ ] ~~Once done all tasks above, get checked by ~~Piers~~ then Adam~~
```

### PR DESCRIPTION
**Pull request by PothieuG**

### CODE REVIEW

**Review by bradystroud — CHANGES_REQUESTED** [review]
As per our conversation with @pierssinclairssw 

1. Line up these list items
image

2. Move the menu item to 
Training | SSW Internship Programs | Full stack Developer
image



image

3. use transparent icon
4. grey text is weird


5. Change from
> Full material access

to
> Training in the latest tech

File: content/training/fullstack.mdx [hunk 1]
```diff
@@ -100,7 +106,7 @@ trainingHeader:
   person: /images/piers.png
 _body:
   - trainingInformationItems:
-      - header: <span class="text-sswRed">Who</span> should apply?
+      - header: <span class="text-sswRed text-5xl">Who</span> should apply?
```
Comment (bradystroud) [inline content/training/fullstack.mdx:109]: Why was this added?

Comment (PothieuG) [reply content/training/fullstack.mdx:109]: I have added the `text-5xl` class because there is a desire to have the first letter in red and larger than the others.

Comment (bradystroud) [reply content/training/fullstack.mdx:109]: Who asked for this?

**Comment by wicksipedia** [comment]
It looks like this PRS, touching way too many files and reverting some changes we made for SEO (changing ssw.com.au to www.ssw.com.au)

File: content/training/fullstack.mdx (hunk 1 above)
Comment (wicksipedia) [reply content/training/fullstack.mdx:109]: It’s also changing the whole word, not the first letter

Comment (PothieuG) [reply content/training/fullstack.mdx:109]: Yes, I apologize, I meant "the first word" not "the first letter" on my comment.
You're correct Brady, I misread the sentence _"Increase font size for the "Who, What, When" section"_ (on the issue). I only enlarged the first words, not the entire section.

**Comment by PothieuG** [comment]
@bradystroud 
@wicksipedia 's comment made me realize that I must have missed something during the merge of master into my branch.
Consequently, I've created a new, clean branch here : https://github.com/SSWConsulting/SSW.Website-v3/pull/686 and I'm closing this one.
